modules/openedx/wordpress/outputs.tf

```diff
@@ -5,3 +5,7 @@ output "rds" {
 output "security_group" {
   value = aws_security_group.wordpress
 }
+
+output "security_group_id" {
+  value = aws_security_group.wordpress.id
+}
```

Comment:
This is not necessary. If you need to access the security group id from outside the module, you can simply do `module.wordpress.security_group.id`

Reply:
@pcockwell This is a rebase artifact. I'll remove it.